pyvista/core/grid.py

```diff
@@ -764,3 +764,41 @@ def gen_coords(i):
         grid.field_data.update(self.field_data)
         grid.copy_meta_from(self, deep=True)
         return grid
+
+    @property
+    def extent(self) -> tuple:
+        """Return or set the extent of the UniformGrid.
+
+        This is the same as :class:`Grid.dimensions` with the upper bounds
+        minus one.
+
+        Examples
+        --------
+        Create a ``UniformGrid`` and show its extent.
+
+        >>> import pyvista
+        >>> grid = pyvista.UniformGrid(dims=(10, 10, 10))
+        >>> grid.extent
+        (0, 9, 0, 9, 0, 9)
+
+        >>> grid.extent = (2, 5, 2, 5, 2, 5)
+        >>> grid.extent
+        (2, 5, 2, 5, 2, 5)
+
+        Note how this changes the bounds as well. Since we use default spacing
```

Comment:
Adding this example is helpful. Thanks! 

Suggestion (a bit more comprehensible): 

_Note how this also modifies the grid bounds and dimensions. Since we use default spacing of 1 here, the bounds_ match the extent exactly.

Reply:
Done in [e6943fc](https://github.com/pyvista/pyvista/pull/2827/commits/e6943fc80c296e43f779362782210ff24b6e36aa).